Tasks/ManualIntervention/task.json

```diff
@@ -30,6 +30,14 @@
                 "rows": "10",
                 "maxLength": "500"
             }
+        },
+        {
+            "name": "emailRecipients",
+            "type": "identities",
```

Comment:
Is identities a new type introduced? I don't see any other task using this type. All others have been using string for identity input.